PinpointKit/PinpointKit/Sources/PinpointKit.swift

```diff
@@ -41,7 +41,8 @@ public class PinpointKit {
      - parameter delegate:           A delegate that is notified of significant events.
      */
     public convenience init(feedbackRecipients: [String], delegate: PinpointKitDelegate? = nil) {
```

Comment:
As a _convenience_ initializer. I can see this being done a few different ways. We could:

- add two parameters with default values for title / body (`"Bug Report"` and `nil`, respectively).
- add a second convenience initializer that takes a `FeedbackConfiguration`
- as you said, replace `feedbackRecipients` with a `FeedbackConfiguration` in this convenience initializer

I kinda like the first, to keep this convenience initializer as convenient as possible, despite a bit of duplication in the default title. I’m happy to implement any, though. Can you let me know if you still prefer the third option after being presented with these?